app/src/main/java/edu/wpi/first/shuffleboard/app/PreloaderController.java

```diff
@@ -22,13 +32,31 @@
 
   @FXML
   private void initialize() {
+    // Bring the hexagons to the top edge to avoid weird blank spots
+    backgroundContainer.setTranslateY(-HEXAGON_RADIUS);
+
     progressBar.setProgress(-1);
     versionLabel.setText(Shuffleboard.getVersion());
+
+    // Hexagon grid background
+    Random random = new Random(System.currentTimeMillis() ^ (System.currentTimeMillis() >> 16));
```

Comment:
Just curious why you are using a custom seed...

Reply:
Because I took this straight from GRIP and didn't change it